### DISCUSSION TITLE
Computing hashes of an already processed evidence

### DISCUSSION THREAD

**Discussion by hpcnetworks** [sepinf-inc/IPED#1951] [Q&A]
HI there!

I've processed an evidence with forensic profile, but somehow the hashes were not calculated.
I'd like to know if there is a way to to process it again only adding the hash calculation, once the evidence has hundreds of Gb...

I tryied to figure out how to perform it, but no success: continue and restart options are intended to other proposes...

Best regards and many thanks, in advance.

Bruno Costa

**Comment by lfcnassif** [comment]
That is not possible until today, being the biggest limitation of the tool, in my opinion. It is a long waiting feature to be implemented:
#24

Unfortunately you will have to process the evidence from scratch.
